examples/http/aws_cloudwatch_metric.js

```diff
@@ -0,0 +1,292 @@
+/**
+ * The driver is designed to retrieve CloudWatch metrics for various AWS services, including EC2, RDS, ELB, and Lambda. 
+ * The metrics that can be monitored include CPUUtilization, NetworkIn, NetworkOut, DiskReadOps, DiskWriteOps, DatabaseConnections, and many more.
+ * 
+ * For a full list of metrics supported by AWS CloudWatch, please refer to the AWS CloudWatch Metrics documentation.
+ * https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/aws-services-cloudwatch-metrics.html
+ * 
+ * The driver uses the script item to make HTTP requests to the CloudWatch API.
+ * Communication protocol is https
+ */
+
+var accessKey = D.device.username(); //accessKey == username
+var secretKey = D.device.password(); //secretKey == password
+var region = "us-east-2"; // change this by you aws region
+var requestPeriod = 600;
+var metrics;
+
+var table = D.createTable("ClouWatch", [
+    { label: "Namespace" },
+    { label: "Dimension" },
+    { label: "Dimension Value" },
+    { label: "Metric" },
+    { label: "Period" },
+    { label: "Stat" },
+    { label: "Value" },
+    { label: "Unit" }
```

Comment:
Is there a need for a unit column considering the column headers accept a unit parameter?